docs/security/access-control.md

```diff
@@ -0,0 +1,646 @@
+---
+title: "Access Control"
+slug: /security/access-control
+keyword: security
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Overview
+
+Gravitino adopts RBAC and DAC. 
+
+Role-based Access Control (RBAC): Access privileges are assigned to roles, which are in turn assigned to users or groups.
+
+Discretionary Access Control(DAC): Each metadata object has an owner, who can in turn grant access to that object.
+
+:::info
+
+Gravitino only supports authorization for secureable objects, when it comes to authentication.
```

Comment:
What's the meaning of "when it comes to authentication", this sentence is hard to understand.

Reply:
cc @yuqi1129

Reply:
What I want to express:  the second sentence `Gravitino doesn't support metadata authentication.` is about authentication because the first sentence is just about `authorization`. The sentences in this passage are unnaturally connected.